package.json

```diff
@@ -45,7 +45,7 @@
     "lint-shell": "find ./src ./tests ./scripts -type f -regextype posix-extended -regex '.*\\.(sh)' -exec shellcheck {} +",
     "docs": "shx rm -rf ./docs && typedoc --gitRevision master --tsconfig ./tsconfig.build.json --out ./docs src",
     "bench": "rimraf ./benches/results && tsx ./benches/index.ts",
-    "postinstall": "patch-package"
+    "pretest": "patch-package"
```

Comment:
Isn't the point of this is to patch the fast check? This would run on every test command. That doesn't make sense. It should be done on installation?

Reply:
See below comment.